zones/reader.go

```diff
@@ -107,6 +107,9 @@ func (zone *Zone) ReadZoneFile(fileName string) (zerr error) {
 
 		case "data":
 			data = v.(map[string]interface{})
+
+		case "parseIP":
+			zone.ParseIP = v.(bool)
```

Comment:
It will cause panic if incorrect conf json (v NOT bool type).
Let's consider handle exception graceful and prompt explicit error msg.

Reply:
I think this error will get handled in [https://github.com/nknorg/geodns/blob/086ba735d6bd0f24871c716aea6e057228d1db82/zones/muxmanager.go#L134](url)

Reply:
What he meant is that we should use
`zone.ParseIP, ok = v.(bool)`
instead of
`zone.ParseIP = v.(bool)`
The first one will not panic but the second one will panic if v is not a bool

Reply:
OK. The panic had recover by https://github.com/nknorg/geodns/commit/f8d2807a7971455dc16ad440b2f5153c7877cd7d#diff-d04a8ce94a0b5747dc67fc561301bf73L107